test/client_test.js

```diff
@@ -15,36 +14,40 @@ if (!PULSE_CONNECTION_STRING) {
 }
 
 suite('buildConnectionString', function() {
```

Comment:
this is now testing `pulseCredentials`, so it should probably be in `test_credentials.js`.